crates/burn-tch/README.md

```diff
@@ -243,15 +243,32 @@ is to use a PyTorch installation. This requires a Python installation.
 _Note: MPS acceleration is available on MacOS 12.3+._
 
 ```shell
-pip install torch==2.2.0
+pip install torch==2.2.0 numpy==1.26.4 setuptools
```

Comment:
Why the added explicit dependencies for numpy and setuptools? 🤔

Reply:
When I preparing the environment, I just `pip install torch==2.2.0`. When `cargo run`, an error raised saying `need numpy`, then I installed `numpy`, it then said need `numpy<2`, then I reinstalled the newest 1.x numpy. It said need `setuptools`. After that, it compiled.

I work on macOS and `venv`, and `pip`(24.2) will not install the dependencies automatically.

Reply:
Huh, that's weird. Dependencies are usually automatically installed..

I don't have a mac to test so I believe you. Let's leave the additional dependencies in the instructions in this case 👍